README.md

```diff
@@ -38,10 +38,15 @@ When running the demo some an `values.yaml` file is created as `.demo/values.yam
 This file can be used with helm to interact with the running demo to make changes to what is running in the cluster.
 
 ```bash
+export RABBITMQ_PASSWORD=$(kubectl get secret --namespace "default" rabbitmq-secret -o jsonpath="{.data.rabbitmq-password}" | base64 -d)
+export MYSQL_ROOT_PASSWORD=$(kubectl get secret --namespace "default" mysql-secret -o jsonpath="{.data.mysql-root-password}" | base64 -d)
+export MYSQL_PASSWORD=$(kubectl get secret --namespace "default" mysql-secret -o jsonpath="{.data.mysql-password}" | base64 -d)
+
 # Show what will be changed by running "helm upgrade"
-helm diff upgrade diracx-demo ./diracx --values .demo/values.yaml
+helm diff upgrade diracx-demo  ./diracx --values .demo/values.yaml --set rabbitmq.auth.password=$RABBITMQ_PASSWORD  --set mysql.auth.rootPassword=$MYSQL_ROOT_PASSWORD --set mysql.auth.password=$MYSQL_PASSWORD
+
 # Actually run "helm upgrade" to apply changes
-helm upgrade diracx-demo ./diracx --values .demo/values.yaml
+helm upgrade diracx-demo ./diracx --values .demo/values.yaml --set rabbitmq.auth.password=$RABBITMQ_PASSWORD  --set mysql.auth.rootPassword=$MYSQL_ROOT_PASSWORD --set mysql.auth.password=$MYSQL_PASSWORD
```

Comment:
We used to be able to avoid the need to set values extracted from the cluster to upgrade

Reply:
Hum, I do not think I have changed this logic at all. Definitely not for `RabbitMQ`. But it is very possible I ended up in a weird state at some point, I'll check.

Reply:
Seems like it still work for performing the actual upgrade, but `diff upgrade` complains about it.

Reply:
Yep https://github.com/databus23/helm-diff/issues/460

Reply:
I've updated the comments and the commands